### ISSUE TITLE
Help text for new Stratis pool "Manage filesystem sizes" option

### ISSUE THREAD

**Issue by briansmith0** [cockpit-project/cockpit#19461]
### Explain what happens

Using Cockpit 300 create a new Stratis pool.  Click the question box next to the "Manage filesystem sizes" check box, and the following text is shown:

_When this option is checked, the new pool will not allow overprovisioning. You need to specify a maximum size for each filesystem that is created in the pool. Filesystems can not be made larger after creation. Snapshots are fully allocated on creation. The sum of all maximum sizes can not exceed the size of the pool. The advantage of this is that filesystems in this pool can not run out of space in a surprising way. The disadvantage is that you need to know the maximum size for each filesystem in advance and creation of snapshots is limited._

My understanding is that when using the no-overprovision mode the filesystem will automatically grow when it reaches 50% utilization, so the "Filesystems can not be made larger after creation" is not correct.  

### Version of Cockpit

300

### Where is the problem in Cockpit?

Storage

### Server operating system

None

**Comment by mvollmer** [comment]
> My understanding is that when using the no-overprovision mode the filesystem will automatically grow when it reaches 50% utilization, so the "Filesystems can not be made larger after creation" is not correct.

Yes, I have learned that as well now.  This text was removed in https://github.com/cockpit-project/cockpit/pull/20611
